scripts/cc-cli/src/commands/send.ts

```diff
@@ -32,7 +24,7 @@ async function sendAction(options: OptionValues) {
   checkAddress(options.to, api);
 
   // Build account
-  const seed = getSeedFromOptions(options);
+  const seed = await getStashSeedFromEnvOrPrompt();
```

Comment:
I think this is similar to `distributeRewards` where anyone can send funds so maybe use `CC_SEED`.

Reply:
100%

Reply:
done